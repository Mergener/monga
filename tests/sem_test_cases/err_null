type intarray = [int];

function main() {
    var i : int;
    var is : intarray;
    is = null;
    i = null;
}