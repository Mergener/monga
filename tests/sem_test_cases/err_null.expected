[Error] At line 7, column 14: Cannot assign value of type '<null-type>' to lvalue of type 'int'.
Semantic errors were caught during analysis.
